webapp/src/containers/Rhs/projectDetails/header.tsx

```diff
@@ -58,22 +63,43 @@ const Header = ({projectDetails, showAllSubscriptions, handlePagination, setShow
 
     // Handles unlinking a project and fetching the modified project list
     const handleConfirmUnlinkProject = () => {
-        makeApiRequestWithCompletionStatus(pluginConstants.pluginApiServiceConfigs.unlinkProject.apiServiceName, projectDetails);
+        makeApiRequestWithCompletionStatus(pluginConstants.pluginApiServiceConfigs.unlinkProject.apiServiceName, {...projectDetails, deleteSubscriptions} as APIRequestPayload);
+    };
+
+    // Update the modal when project unlinking fails
+    const handleActionsAfterUnlinkingProjectFailed = (err: ApiErrorResponse) => {
+        const errorMessage = utils.getErrorMessage(true, 'ConfirmationModal', err);
+        if (errorMessage === pluginConstants.messages.error.adminAccessError) {
+            setConfirmationModalDescription(pluginConstants.messages.error.adminAccessErrorForUnlinking);
```

Comment:
Is this correct that you are showing the error In the description of the modal. Is there a need to create an error state for this.

Reply:
No we need to display the message in the description. This will be displayed when a non admin user trie to delete a subscription. In such case we want the user to know that u cannot delete the subscription but can unlink the project